platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -1641,7 +1641,7 @@ Ref<Image> DisplayServerX11::screen_get_image(int p_screen) const {
 			ERR_FAIL_V_MSG(Ref<Image>(), vformat("XImage with RGB mask %x %x %x and depth %d is not supported.", (uint64_t)image->red_mask, (uint64_t)image->green_mask, (uint64_t)image->blue_mask, (int64_t)image->bits_per_pixel));
 		}
 		img = Image::create_from_data(width, height, false, Image::FORMAT_RGBA8, img_data);
-		XFree(image);
```

Comment:
There's another `XFree(image)` a few lines above.

Reply:
That part actually looks weird, why is it using image right after it was freed? I'm not sure what the original intent wasl but I think it's safe to just remove that line.

Reply:
It's freeing before returning (`ERR_FAIL_V_MSG` returns).

Reply:
I see, but is it alright that `ERR_FAIL_V_MSG` itself refers to image, like `image->red_mask`?

Reply:
No, that's another bug indeed. I would suggest saving the error message as a `String` before freeing `image`, and then pass it to `ERR_FAIL_V_MSG`.